src/components/App/App.js

```diff
@@ -2,34 +2,52 @@ import React, { Component } from 'react';
 import Header from '../Header/Header';
 import Movies from '../Movies/Movies';
 import Film from '../Film/Film';
-import movieData from './movie-data';
 import './App.css';
 
 class App extends Component {
   constructor() {
     super();
     this.state = {
-      movies: movieData.movies,
-      currentMovie: {},
-      isHome: true
+      movies: [],
+      currentMovie: '',
+      isHome: true,
+      isLoading: true,
+      error: false
+    }
+  }
+
+  componentDidMount() {
+    fetch('https://rancid-tomatillos.herokuapp.com/api/v2/movies/')
+      .then(response => response.json())
+      .then(films => this.checkForError(films))
+      .catch(error => this.setState({isLoading:false, error: true}))
+  }
+
+  checkForError = films => {
+    if(films.error) {
+      this.setState({error: true, isLoading: false})
+    } else {
+      this.setState({movies: films.movies, isLoading: false})
     }
   }
 
   goHome = () => {
-    this.setState({ currentMovie: {}, isHome: true})
+    this.setState({ currentMovie: '', isHome: true})
   }
 
-  selectMovie = () => {
-    this.setState({ currentMovie: {}, isHome: false });
+  selectMovie = (event) => {
```

Comment:
Maybe it would be better to target the card by it's classname rather than the tag name in the event target